tests/mypy_test.py

```diff
@@ -21,53 +21,55 @@
 from typing_extensions import Annotated, TypeAlias
 
 import tomli
-from utils import colored, print_error, print_success_msg, read_dependencies
+from utils import VERSIONS_RE as VERSION_LINE_RE, colored, print_error, print_success_msg, read_dependencies, strip_comments
 
-SUPPORTED_VERSIONS = [(3, 11), (3, 10), (3, 9), (3, 8), (3, 7)]
+SUPPORTED_VERSIONS = ["3.11", "3.10", "3.9", "3.8", "3.7"]
```

Comment:
Any reason not to centralize this in utils.py? It would be nice to minimize the number of places we have to change when 3.12 comes around or when we drop 3.7.

Reply:
Yup, definitely. I'd like to do something like that in a follow-up PR, though — I have a few more ideas about things I'd like to do in that vein :)